src/ruby.rs

```diff
@@ -947,6 +947,22 @@ extern {
     ///     [variable.c](https://github.com/ruby/ruby/blob/v2_5_1/variable.c#L459-L463)
     /// * **2.6:** [ruby.h](https://github.com/ruby/ruby/blob/v2_6_0_preview2/include/ruby/ruby.h#L1808)
     pub fn rb_obj_classname(obj: VALUE) -> *const c_char;
+
+    /// Equivalent of `Kernel#block_given?`.
+    /// Returns 1 (true) if yield would execute a block in the current context, or 0 (false)
+    /// otherwise.
```

Comment:
This is probably a bit of a nit-pick, but I think I'd rather do have it this way:

```
    /// Returns 1 (true) if yield would execute a block in the current context, or 0 (false)
    /// otherwise.
    ///
    /// This is the equivalent of `Kernel#block_given?`.

```

We sort of want to get right to the point about what it really does here. The fact that it is analogous to a Ruby method is worth knowing, but doesn't actually tell us how it works.

Reply:
Interestingly, the inline docs on the method say "non-zero" if there's a block, though it is clearly returning `1`.
https://github.com/ruby/ruby/blob/v2_5_1/eval.c#L831

Reply:
"Returns a C boolean (zero if false, non-zero if true)..."